Sources/Controllers/SpotifyController.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  SpotifyController.swift
+//  ControlKit
+//
+
+import OSLog
+import SpotifyiOS
+import SwiftUI
+
+/// Wrapper for ``SPTAppRemote``  - use this to control playback and subscribe to the playback state of the Spotify app.
+///
+/// `SpotifyController` facilitates the connection to and interaction with the Spotify app, enabling playback control and handling the authorization flow.
+///
+/// - Parameters:
+///   - config: The `SpotifyConfig` object used to configure the controller.
+///   If `config` is `.empty`, the controller will not attempt to connect automatically.
+///   The default log level is ``SPTAppRemoteLogLevel//debug``.
+///   - accessTokenDAO: An object conforming to `DAO<String>` that manages the storage and retrieval of the Spotify access token. By default, this uses a `UserDefaultsDAO` with a key of `"SpotifyAccessToken"`.
+///   - autoConnect: A Boolean value that determines whether the controller should attempt to connect to Spotify automatically upon initialization.
+///   The default value is `true`. If `true` and `config` is not `.empty`, the controller will initiate the connection.
+///
+/// - Warning: Saving the Spotify access token using **``UserDefaults``** is not recommended for production apps 🚨
+///   Prefer a `DAO` that persists values to the Keychain 🔐
```

Comment:
I guess it can be added as a description for the init method with the note that `UserDefaultsDAO("SpotifyAccessToken")` can be used for testing.

Reply:
Sounds good, I've split this documentation across the struct declaration + init method 🤝